lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -286,6 +287,27 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL
+   * Collapsed content is opened automatically
+   * Element is highlighted via CSS selector
+   */
+  _jumpToAudit() {
+    const hash = window.location.hash;
+    if (hash) {
```

Comment:
nit: WDYT about reversing these conditions to avoid nesting?

```js

if (!hash) return;

const audit = ...
if (!audit) return;

...

Reply:
Good one. Heavy nesting is ugly indeed^^